packages/create-astro/README.md

```diff
@@ -42,6 +42,7 @@ May be provided in place of prompts
 |:-------------|:----------------------------------------------------|
 | `--template` | Specify the template name ([list][examples])        |
 | `--commit`   | Specify a specific Git commit or branch to use from this repo (by default, `main` branch of this repo will be used) |
+| `--fancy`    | For Windows users, `--fancy` will enable full unicode support |
```

Comment:
Did we have a `--help` flag that would print this stuff in the CLI in a previous version? Out of scope for this PR, but could be helpful to add in the future.

Docs here look good though! Even if I guess this README doesn’t get a lot of eyes.

Reply:
I don't think we do have a `--help` flag, would be nice to follow-up with!